packages/components/src/lib/selection/media/html-selector.js

```diff
@@ -256,8 +257,8 @@ export default class HTMLSelector extends MediaSelector {
     }
     // clean string:
     //   convert punctuation to spaces
-    anchorText = anchorText.replace(new RegExp('[' + textSelector.model.getPunctuation() + ']', 'g'), ' ')
 
+    anchorText = anchorText.replace(new RegExp('[' + textSelector.model.getPunctuation() + ']+', 'g'), ' ')
```

Comment:
this is the fix for 
> I did also notice that the two digit words are not always selected on mousemove: sometimes it just ignored it, sometimes it does highlighte the line and generate a wrong pop-up, sometimes it does process the two digit word.
see examples below in Google doc

the problem here that, Google docs converts spaces to special characters, 
and they are converted with this Regexp to several spaces, that could braek words definition